introduction.md

```diff
@@ -1,4 +1,7 @@
 Introduction
 ============
 
-Hi, I'm osserpse! My favorite Code School paths is Javascript.
\ No newline at end of file
+Hi, I'm osserpse! My favorite Code School paths is Javascript.
+
+* Git
+* Javascript
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?